3 - PB/Documentazione esterna/Manuale Utente/Manuale Utente.typ

```diff
@@ -84,6 +84,70 @@ Per i requisiti di sistema e hardware si rimanda al documento #st_v.
 
 
 = Istruzioni d'uso
+== Avvio e Configurazione dell'ambiente
+
+Il software all'avvio si presenta come segue:
+#figure(
+  image("./imgs/avvio.png", width: 60%),
+  caption: [
+    Avvio del software
+  ],
+) <avvio>
+qui vengono proposte due diverse configurazioni iniziali possibili, *Planimetria rettangolare* e *Planimetria personalizzata*. Le differenze tra le due modalità di lavoro verranno descritte nei capitoli dedicati.
+
+Si può scegliere una modalità di lavoro selezionandola nel pannello di @avvio e premendo il pulsante *Prossimo* in basso a destra.
+
+=== Inizializzazione Planimetria Rettangolare
+
+#figure(
+  image("./imgs/planimetria_rettangolare.png", width: 60%),
+  caption: [
+    Definizione parametri del piano rettangolare
+  ],
+) <piano_rettangolare>
+
+Selezionata la Planimetria Rettangolare nella schermata di @avvio il sistema mostrerà la schermata mostrata in @piano_rettangolare, dove è possibile definire la lunghezza e la profondità del magazzino che vogliamo creare.
+
+Per terminare la configurazione del piano è richiesto che i due parametri siano entrambi maggiori di 0, in caso contrario verrà impedito di procedere alla visualizzazione del piano e verrà visualizzato un errore.
+
+Spuntando invece le checkbox "importa i prodotti dal database" il sistema, terminata la configurazione dell'ambiente, caricherà nell'apposita finestra
+*Product* i prodotti presenti nel database.
+
+Premere quindi il pulsante *Submit* per procedere alla visualizzazione dell'ambiente 3D, premere il pulsante *Indietro* per annullare la configurazione e tornare al menù mostrato in @avvio
+
+=== Inizializzazione Planimetria Personalizzata
+
+#figure(
+  image("./imgs/planimetria_personalizzata.png", width: 50%),
+  caption: [
+    Definizione parametri del piano personalizzato
+  ],
+) <piano_personalizzato>
+
+Selezionata la Planimetria Personalizzata nella schermata di @avvio il sistema mostrerà la schermata mostrata in @piano_personalizzato, dove è possibile caricare il file SVG che verrà disegnato sul piano e la misura del lato maggiore in modo da scalare correttamente la piantina che verrà disegnata.
```

Comment:
Cambierei il paragrafo in

"Selezionata l'opzione "Planimetria personalizzata" nella schermata di `@avvio`, verrà mostrata la schermata presente in `@piano_personalizzato`, dove sarà possibile caricare il file SVG che verrà disegnato sul piano, e inserire la misura del lato maggiore del magazzino, in modo da scalarne correttamente la pianta."